src/InlineRelationsTransformer.cpp

```diff
@@ -795,7 +795,7 @@ NullableVector<std::vector<AstLiteral*>> getInlinedLiteral(AstProgram& program,
                 }
             }
         }
-    } else if (auto* neg = dynamic_cast<AstNegation*>(lit)) {
+    } else if (AstNegation* neg = dynamic_cast<AstNegation*>(lit)) {
```

Comment:
The type is already specified in the dynamic cast, so `auto` is simpler and equally clear.